R/update_package_object.R

```diff
@@ -0,0 +1,59 @@
+#' Update a data object and accompanying package resource map
+#'
+#' This function updates an data object, and then automatically
+#' updates the package resource map with the new data pid. It
+#' is a convenience wrapper around \code{\link{arcticdatautils::update_object}}
```

Comment:
Does `\code{\link{arcticdatautils::update_object}}` make the link work right in the documentation? Links to other packages are usually done like `\link[PKG]{fn}`.

Reply:
don't know!  I was guessing a bit on that based on what I saw in the arcticdatautils code.  I'll change it.

Reply:
That's some egg on my face there. Sorry about that!